uiowa/uiowa
# PDF styles.

Resolves #2563 

### Testing
- `yarn workspace admissions_core run build`
- `blt sync --site admissions.uiowa.edu`
- Go to https://admissions.local.drupal.uiowa.edu/areas-of-study/print-links and ensure access is denied while logged out.
- Log in and revisit link above as viewer or other role and ensure access granted.
- Click Accounting from the print links view page and ensure PDF viewer renders.
- Edit the Accounting AOS and add a value to the Mail Code field.
  - http://admissions.local.drupal.uiowa.edu/areas-of-study/accounting
- Go back to the PDF viewer for accounting and download PDF and ensure filename is set to the Mail Code value entered.
  - http://admissions.local.drupal.uiowa.edu/print/pdf/node/761 
- Review PDF display, styles and templates. See #2970 for instructions on working with SASS/template changes.
  - http://admissions.local.drupal.uiowa.edu/print/pdf/node/761/debug
  - Add a really long page title and test that it wraps within the pdf
